.idf_build_apps.toml

```diff
@@ -1,5 +1,3 @@
-target = "all"
-paths = "."
```

Comment:
Were these changes necessary for some reason? They are part of a commit titled "review changes", so hard to understand why they are removed.

Reply:
With this change you can use `idf-build-apps` also locally eg `idf-build-apps find -t esp32s2 -p some_path`

Without the change, the CLI options -t and -p are overriden byt the .toml file

Reply:
Originally I added the 
```
target = "all"
paths = "."
```
part because in some older versions of idf-build-apps this was not default behavior